website/snippets/_sl-test-and-query-metrics.md

```diff
@@ -1,13 +1,23 @@
-This section will explain how you can locally test and query metrics. Before you begin, refer to [MetricFlow CLI](/docs/build/metricflow-cli) for instructions on installing it and a reference for the CLI commands.
+:::important Testing and querying metrics in the dbt Cloud IDE not yet supported
 
-:::important
+For public beta, testing or querying metrics in the dbt Cloud IDE isn't supported yet, however support is coming soon. 
```

Comment:
It might read better if it's more concise. 
```suggestion
Support for testing or querying metrics in the dbt Cloud IDE is not available in the current beta but is coming soon. 
```